dsl/ChalkODMProp.h

```diff
@@ -7,38 +7,39 @@ namespace sorbet::dsl {
 /**
  * This class desugars things of the form
  *
- *   prop :foo, String
+ *   prop :foo, Type
  *
  * into
  *
- *   sig {returns(T.nilable(String))}
- *   def foo; T.cast(nil, T.nilable(String)); end
- *   sig {params(arg0: String).returns(NilClass)}
- *   def foo=(arg0); end
+ *   sig {returns(Type)}
+ *   def foo; ...; end
+ *   sig {params(x: Type).returns(Type)}
```

Comment:
https://github.com/sorbet/sorbet/blob/23395fe680e03f84c76ccdd6faa67094d9dcf862/dsl/helpers.cc#L18-L19

^ it does use `arg0` for the setter's argument name